src/domain/entities/ProgramAttributes.ts

```diff
@@ -0,0 +1,15 @@
+import { Id } from "./Base";
+
+export type ProgramAttributes = {
+    id: Id;
+    attributes: Attribute[];
+    orgUnit: Id;
+    trackedEntityType: Id;
+    programId: Id;
+};
+
+export type Attribute = {
```

Comment:
let's call that AttributeValue to differentiate the attribute itself from the value in a TEI.